src/components/table/Table.js

```diff
@@ -668,12 +668,12 @@ class Table extends Component {
 
                     <div
                         className={
-                            "panel panel-primary panel-bordered panel-bordered-force table-flex-wrapper document-list-table" +
+                            "panel panel-primary panel-bordered panel-bordered-force table-flex-wrapper document-list-table " +
                             ((
                                 (rowData && rowData[tabid] &&
                                 Object.keys(rowData[tabid]).length === 0) ||
                                 (!rowData[tabid])
-                            ) ? " table-content-empty" : null)
```

Comment:
@chmieludaniel If you are toggling css class:
- please, keep consistent the project schema, always the one free space char is *after* toggled class
- return empty string instead of null